draft-terbu-sd-jwt-vc.md

```diff
@@ -31,8 +31,9 @@ organization="Authlete Inc. "
 .# Abstract
 
 This specification describes data formats as well as validation and processing
-rules to express Verifiable Credentials with JSON payload based on the SD-JWT
-format [@!I-D.ietf-oauth-selective-disclosure-jwt].
+rules to express Verifiable Credentials based on the Selective Disclosure
+for JWT and JWS with JSON payloads (SD-JWT) format [@!I-D.ietf-oauth-selective-disclosure-jwt].
+It can be used when there are no selective disclosable claims, too.
```

Comment:
```suggestion
```

Reply:
I agree that the supported securing mechanisms should be explicit.

Should the specification define which claims can be hidden and which not when SD is applied? e.g., iss, exp, nbf, cnf is probably conditional.

Reply:
This is already the case.
> If present, the following registered JWT claims MUST be included in the SD-JWT and MUST NOT be included in the Disclosures, i.e. cannot be selectively disclosed:

Reply:
I don't think it needs to be highlighted in the introduction. Rationale is sufficient in my opinion.

Reply:
in other issue.